casaos-healthcheck/run.sh

```
@@ -140,23 +140,152 @@ check_dns_resolution() {
+            fi
+        else
+            print_color "0;33" "${WARNING_MARK} Unable to check health of /dev/$disk"
         fi
     done
 }
 
+# Function to check disk space
+check_disk_space() {
+    print_header "Disk Space Check"
+    local threshold=80
+    local usage=$(df / | grep / | awk '{ print $5 }' | sed 's/%//g')
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)